jp/lessons/basics/basics.md

```diff
@@ -6,28 +6,46 @@ order: 1
 lang: jp
 ---
 
-セットアップ、基本型、そして演算。
+入門、基本データ型、そして基本的な演算。
 
 {% include toc.html %}
 
-## セットアップ
+## 入門
 
 ### Elixirのインストール
 
 各OS向けのインストール方法は Elixir-lang.org 上の[Installing Elixir](http://elixir-lang.org/install.html) で探すことができます。
 
+Elixirがインストールされたら簡単にバージョンを確認できます。
+
+    % elixir -v
+    Erlang/OTP 18 [erts-7.3] [source] [64-bit] [smp:4:4] [async-threads:10] [hipe] [kernel-poll:false] [dtrace]
```

Comment:
@nscyclone could you explain to @hinamiyagk what needs to be changed here to support your last PR?